datumaro/components/operations.py

```diff
@@ -1292,7 +1292,7 @@ def _default_hash(item):
             log.warning("Item (%s, %s) has no image "
                 "info, counted as unique", item.id, item.subset)
             return None
-        return hashlib.md5(item.image.data.tobytes()).hexdigest()
+        return hashlib.md5(item.image.data.tobytes()).hexdigest() # nosec - disable B303:md5 check
```

Comment:
Probably, I'd better have comments about the reasons to ignore, instead of ignored rules. 

```suggestion
        # ignore B303 (md5 check), because it is not used in security context
        return hashlib.md5(item.image.data.tobytes()).hexdigest() # nosec
```

Reply:
Why not just change it to use a non-obsolete hash algorithm?

Reply:
We need any fast hashing algo here, which one would you suggest?

Reply:
SHA-256? 🤷

Reply:
Looking at https://stackoverflow.com/questions/10070293/choosing-a-hash-function-for-best-performance, I can conclude, that md5 is a good baseline. There are few other algos, which up to 2 times faster, everything else is worse. CRC32 might be good candidate, but it has worse entropy, so we can expect more collisions.

Reply:
Okay, fair enough.